src/evergreen/__init__.py

```diff
@@ -16,5 +16,5 @@
 from evergreen.task_reliability import TaskReliability
 from evergreen.version import Version
 
-VERSION = (0, 6, 14)
+VERSION = (0, 6, 15)
```

Comment:
Could you also update `CHANGELOG.md` with a description of this change?

Reply:
done